.github/workflows/release.yml

```diff
@@ -0,0 +1,68 @@
+name: Sync and Create PRs
```

Comment:
This will update go-primitives and create PRs at related repos, will save some time in further

Reply:
I replied in the Element thread, I suggest to add opBNB first in this PR, and create another pr for automating the process.

Reply:
@iCell Sure, I removed this workflow from here. Please review